tests/test.config.ts

```diff
@@ -21,10 +21,6 @@ Object.keys(environment).forEach((key) => {
   process.env[key] = environment[key];
 });
 
-// Disable Sentry warnings
-import * as raven from 'raven';
```

Comment:
Is there an alternative for handling the logs during tests with the new package?